devenv.nix

```diff
@@ -19,6 +19,8 @@ in {
   env.GOPATH = config.env.DEVENV_STATE + "/go";
   env.GOTOOLCHAIN = "local";
 
+  apple.sdk = null; # use installed apple sdk, fixes broken nix packages
```

Comment:
Does this fix running `nix build .#checks.aarch64-linux.firewall`?

Reply:
No idea, this is for devenv only so that I can run `go run ui...` against my current macOS. Default SDK is 11, which is too old for anything graphical.

Reply:
The reason for the error I am seeing: https://github.com/tfc/nixos-integration-test-example/issues/1